### DISCUSSION TITLE
Duda login con archivos fiel

### DISCUSSION THREAD

**Discussion by RonaldoNEzaudita** [phpcfdi/cfdi-sat-scraper#82] [Q&A] [answered]
Hola buen dia equipo, tengo una duda si hago login con archivos FIEL llegara en un momento en pedir resolver captcha

**Comment by MiguelAngelMP10** [comment, marked as answer]
Eso no es posible siempre y cuando siempre uses la autenticación con fiel 
[Ejemplo de autenticación con FIEL](https://github.com/phpcfdi/cfdi-sat-scraper?tab=readme-ov-file#ejemplo-de-autenticaci%C3%B3n-con-fiel)

**Comment by eclipxe13** [comment]
Para aclarar la respuesta:
En la autenticación con CIEC se requiere resolver un captcha.
En la autenticación con FIEL no se pregunta un captcha.
